app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    data = command.split()
+    file_to_copy = data[1]
+    copy = data[2]
+
+    if len(data) != 3 or data[0] != "cp" or file_to_copy == copy:
+        return
+
+    with open(file_to_copy, "r") as file_in, open(copy, "w") as file_out:
```

Comment:
Consider adding exception handling for file operations to manage potential errors such as file not found or permission denied. This will make your function more robust.